internal/client/clustergroup/cluster_group_resource.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright © 2021 VMware, Inc. All Rights Reserved.
+Copyright © 2022 VMware, Inc. All Rights Reserved.
```

Comment:
Why is this being updated

Reply:
Missed to revert this while moving iam policy client to separate directories. Changed